### ISSUE TITLE
Class path contains multiple SLF4J bindings

### ISSUE THREAD

**Issue by RDDOUB** [linkedin/WhereHows#153]
Has anyone else run into a similar error, and if so how did you resolve?

SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/ubuntu/WhereHows/backend-service/lib/slf4j-log4j12-1.7.12.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/ubuntu/play-2.2.4/repository/local/ch.qos.logback/logback-classic/1.0.13/jars/logback-classic.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]

**Comment by rkluszczynski** [comment]
@RDDOUB Hello, we had the same issue ending up with problems of missing logs (for scheduled etl job). We have managed to overcome that a week and a half ago. I've created pull request #155 which should solve that problem (I mean it should no longer bind to org.slf4j.impl.Log4jLoggerFactory and always use Logback).

Maybe this will also help you:)

Of course, I could miss some another way of execution which depends on log4j, we will see in comments by linkedin team:) At the moment it works for us.

Kind regards,
Rafal Kluszczynski
